ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -622,8 +622,8 @@ else
     <div class="centerwrapper no-print">
         <div class="inlineblockwrapper">
             <div id="submissionHistory" style="display: @((Model.SubmissionViews.Count == 0) ? "none" : "block")" class="hideablepanel">
-                Click here to view previous submissions: 
-                <div class="panelcontents" style="display: none">
+                Click here to view previous submissions:
```

Comment:
Why does it say "click here" if the submissions can already be viewed?

Reply:
Good point, you can still click to close, but I never change that text depending on whether it's open or closed.